sc/social/like/vocabularies.py

```diff
@@ -46,3 +54,18 @@ def __call__(self, context):
     SimpleTerm(value=u'Like', title=_(u'Like')),
     SimpleTerm(value=u'Share', title=_(u'Share')),
 ])
+
+
+@provider(IVocabularyFactory)
+def ImageScalesVocabulary(object):
+    """Obtains available scales from plone.app.imaging"""
+    terms = []
+    for scale, (width, height) in getAllowedSizes().iteritems():
+        if width >= LEAD_IMAGE_MIN_WIDTH and height >= LEAD_IMAGE_MIN_HEIGHT:
+            translated = _(
```

Comment:
is Plone translating the scales?

Reply:
I got the original. https://github.com/plone/plone.app.vocabularies/blob/2.1.x/plone/app/vocabularies/images.py